dipy/reconst/csdeconv.py

```diff
@@ -118,18 +120,20 @@ def __init__(self, gtab, response, reg_sphere=None, sh_order=8, lambda_=1,
         # scale lambda_ to account for differences in the number of
         # SH coefficients and number of mapped directions
         # This is exactly what is done in [4]_
-        self.lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
-                        (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.))
-                       )
+
+        lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
+                   (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.)))
```

Comment:
I think that it's fine to leave this as is (if only for backward compatibility), but it would be good to add a comment here, for future curious souls (as suggested above by @matthew-brett), just to point out that this is not something that needs to be normalized away, and that users should consider how they set their lambda input, depending on their data/sampling sphere